hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapStoreTest.java

```diff
@@ -358,6 +358,30 @@ public void issue614() {
         assertEquals(0, localMapStats.getDirtyEntryCount());
     }
 
+    @Test
+    public void clear() {
```

Comment:
`clear` is already tested in `testMapstoreDeleteOnClear` test; if the idea is to test the same with an `EAGER` map store, maybe reflect it in the test name?

Reply:
it is left-over, deleted it.